cpp/ql/src/Likely Bugs/Conversion/CastArrayPointerArithmetic.ql

```diff
@@ -64,10 +64,15 @@ predicate introducesNewField(Class derived, Class base) {
   )
 }
 
-from DataFlow::PathNode source, DataFlow::PathNode sink, CastToPointerArithFlow cfg
+pragma[nomagic]
```

Comment:
Do we really want `nomagic` here?  That means we're evaluating it for every `DataFlow::PathNode` when we only need it for the sources and sinks.

Reply:
Sounds like we're doing fine as we are.  Thanks for the extra explanation.